src/main/java/az/company/mssubscription/service/SubscriptionService.java

```diff
@@ -0,0 +1,46 @@
+package az.company.mssubscription.service;
+
+import az.company.mssubscription.dao.entity.SubscriptionEntity;
+import az.company.mssubscription.dao.repository.SubscriptionRepository;
+import az.company.mssubscription.enums.SubscriptionStatus;
+import az.company.mssubscription.exception.NotFoundException;
+import az.company.mssubscription.model.request.CreateSubscriptionRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import static az.company.mssubscription.enums.SubscriptionStatus.EXPIRED;
+import static az.company.mssubscription.mapper.SubscriptionMapper.buildSubscriptionEntity;
+import static az.company.mssubscription.model.constants.ErrorMessages.*;
+
```

Comment:
**Not exist send to ms-product logic**. Send in every method via rabbitmq to ms-product